assignment5/assignment5.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+
+function reader(file) {
```

Comment:
reader seems like a noun, function names are expected to be verbs/actions. Rename it accordingly.